src/os/CMakeLists.txt

```diff
@@ -141,6 +137,21 @@ if(WITH_SPDK)
   endforeach()
 endif(WITH_SPDK)
 
+if(WITH_PMEM)
+  include_directories("${CMAKE_SOURCE_DIR}/src/nvml/src/include")
+  add_custom_target(build_nvml ALL
```

Comment:
@majianpeng if the pmem backend is optional, can we add it as an external project using `ExternalProject_Add()`?

Reply:
Ok , i'll update this.